spotpy/unittests/test_database.py

```diff
@@ -36,6 +36,7 @@ def test_csv_multiline(self):
         csv = db.csv("UnitTest_tmp", self.parnames, self.like, self.randompar, simulations=self.simulations_multi,
                           chains=1, save_sim=True)
 
+        csv.save(self.like ,self.randompar,self.simulations_multi)
```

Comment:
The extra `csv.save` is so that `csvdata` has more than one entry in the list. (It wouldn't make sense to have a single entry in the CSV file anyway, so this doesn't break anything)